Incompatibility with maven 3.3.9

I have just upgraded to dependency-check 7.4.1 to eliminate some false positive.  Running the plugin with maven 3.3.9 I get the same error described in https://github.com/jeremylong/DependencyCheck/issues/1054.  If I use maven 3.5.3 it works fine.
Documentation at https://jeremylong.github.io/DependencyCheck/dependency-check-maven/index.html still says "The plug-in requires Maven 3.1 or higher".  Is this a regression that will be fixed?  If not, please, update the doc with the new minimum maven version.

For documentation: I had the same problem with maven 3.3.9, but works with maven 3.6.3

StackTrace I encountered:
```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:7.4.1:check (default) on project symeda-default-pom: One or more exceptions occurred during dependency-check analysis: One or more exceptions occurred during analysis:
[ERROR] ArtifactResolverException: com.google.inject.ProvisionException: Unable to provision, see the following errors:
[ERROR] 
[ERROR] 1) Error injecting: private org.eclipse.aether.spi.log.Logger org.apache.maven.repository.internal.DefaultVersionRangeResolver.logger
[ERROR] while locating org.apache.maven.repository.internal.DefaultVersionRangeResolver
[ERROR] while locating java.lang.Object annotated with *
[ERROR] at org.eclipse.sisu.wire.LocatorWiring
[ERROR] while locating org.eclipse.aether.impl.VersionRangeResolver
[ERROR] for parameter 1 at org.eclipse.aether.internal.impl.DefaultRepositorySystem.<init>(Unknown Source)
[ERROR] while locating org.eclipse.aether.internal.impl.DefaultRepositorySystem
[ERROR] while locating java.lang.Object annotated with *
[ERROR] Caused by: java.lang.IllegalArgumentException: Can not set org.eclipse.aether.spi.log.Logger field org.apache.maven.repository.internal.DefaultVersionRangeResolver.logger to org.eclipse.aether.internal.impl.slf4j.Slf4jLoggerFactory
[ERROR] at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:167)
[ERROR] at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:171)
[ERROR] at sun.reflect.UnsafeObjectFieldAccessorImpl.set(UnsafeObjectFieldAccessorImpl.java:81)
[ERROR] at java.lang.reflect.Field.set(Field.java:764)
...
```

--- a/README.md
+++ b/README.md
@@ -252,13 +252,13 @@ mvn -s settings.xml install
 Running dependency-check on dependency-check
 --------------------------------------------
 
-Dependency-check references several vulnerables dependencies that are never used
+Dependency-check references several vulnerable dependencies that are never used
 except as test resources. All of these optional test dependencies are included in
 the `test-dependencies` profile. To run dependency-check against itself simple
-exclude the `test-dependencies` profile;
+exclude the `test-dependencies` profile:
 
 ```shell
-mvn org.owasp:dependency-check-maven:aggregate -P-test-dependencies
+mvn org.owasp:dependency-check-maven:aggregate -P-test-dependencies -DskipProvidedScope=true
 ```
 
 Building the documentation

--- a/maven/pom.xml
+++ b/maven/pom.xml
@@ -175,9 +175,6 @@ Copyright (c) 2013 Jeremy Long. All Rights Reserved.
                     <name>releaseTesting</name>
                 </property>
             </activation>
-            <properties>
-                <maven.api.version>3.1.0</maven.api.version>
-            </properties>
             <build>
                 <plugins>
                     <plugin>

--- a/pom.xml
+++ b/pom.xml
@@ -124,8 +124,7 @@ Copyright (c) 2012 - Jeremy Long
         <slf4j.version>1.7.36</slf4j.version>
         <logback.version>1.2.11</logback.version>
         
-        <!-- moved maven.api.version to the profile section -->
-        <!--maven.api.version>3.8.1</maven.api.version-->
+        <maven.api.version>3.1.0</maven.api.version>
         <reporting.checkstyle-plugin.version>3.2.0</reporting.checkstyle-plugin.version>
         <reporting.checkstyle.tool.version>9.3</reporting.checkstyle.tool.version>
         <doxia-module-markdown.version>1.11.1</doxia-module-markdown.version>
@@ -1380,22 +1379,6 @@ Copyright (c) 2012 - Jeremy Long
         </dependency>
     </dependencies>
     <profiles>
-        <profile>
-            <id>standard</id>
-            <!--
-            `releaseTesting` is used in the maven/pom.xml and downgrades the
-            Maven API to 3.1.0 during full refression testing to ensure that
-            no new Maven APIs were used and we maintain compatability with 3.1.
-            -->
-            <activation>
-                <property>
-                    <name>!releaseTesting</name>
-                </property>
-            </activation>
-            <properties>
-                <maven.api.version>3.8.6</maven.api.version>
-            </properties>
-        </profile>
         <profile>
             <id>release</id>
             <activation>

